admin/package-files/docker-plugin/systemd/flocker-docker-plugin.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Flocker Docker Plugin
+Before=docker.service
+
+[Service]
+ExecStart=/usr/sbin/flocker-docker-plugin
+Restart=always
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
Should there also / instead be a 
 * `Requires=docker.service`
 * http://www.freedesktop.org/software/systemd/man/systemd.unit.html#Requires=